SPHINXsys/src/shared/bodies/base_body.cpp

```diff
@@ -184,7 +190,7 @@ namespace SPH
 			if (fabs(phi) < particle_spacing_min_)
 				tagAParticle(i);
 		}
-		std::cout << "Number of surface particles : " << body_part_particles_.size() << std::endl;
+		//std::cout << "Number of surface particles : " << body_part_particles_.size() << std::endl;
```

Comment:
why did you change this?

Reply:
When the surface particles were called each time, it was put out each time step